src/containers/Transactions/Simple/NFTokenCancelOffer.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+
+import Account from '../../shared/components/Account'
+import { SimpleRow } from '../../shared/components/Transaction/SimpleRow'
+import { Amount } from '../../shared/components/Amount'
+
+export interface Props {
+  data: {
+    instructions: {
+      cancelledOffers: {
+        amount: { currency: string; amount: number; issuer: string }
+        offerID: string
+        tokenID: string
+        offerer: string
+      }[]
+    }
+  }
+}
+
+const NFTokenCancelOffer = (props: Props) => {
+  const {
+    data: {
+      instructions: { cancelledOffers },
+    },
+  } = props
+
+  return (
+    <>
+      {cancelledOffers.map(
+        ({
+          amount: { amount, issuer, currency },
+          offerID,
+          tokenID,
+          offerer,
+        }) => (
+          <>
+            <SimpleRow label="Offer ID">
```

Comment:
In #278 I add a way to provide a className to `<SimpleRow>` which will add a css class to the `.value` element.  This will remove the need for an extra div and you can then move data-test up to `<SimpleRow>`